### DISCUSSION TITLE
GLSL shader implementation/integration of Vebro

### DISCUSSION THREAD

**Discussion by Fabsenk5** [rocksdanister/lively#2460] [Ideas]
Since there is no direct GLSL integration, an implementation of the Vebro code into Lively Wallpaper could be a win-win to run shadertoy.com shaders locally and with pause algorithm, which is missing in Vebro (other than that pretty perfect for shader wallpaper use).

https://github.com/bitrate16/Vebro

![image](https://github.com/user-attachments/assets/d23c55c7-b494-4647-ad88-d9ecc97e604f)

**Comment by rocksdanister** [comment]
I don't know modern C++ programming so it will be difficult for me to modify (implementing pause algorithm etc..)

Usually I prefer creating custom shadertoy wallpaper with LivelyProperties:
https://github.com/rocksdanister/rain
